src/System.Device.Gpio/System/Device/Spi/Devices/Windows10SpiDevice.cs

```diff
@@ -127,9 +148,20 @@ public override void TransferFullDuplex(ReadOnlySpan<byte> writeBuffer, Span<byt
                 throw new ArgumentException($"Parameters '{nameof(writeBuffer)}' and '{nameof(readBuffer)}' must have the same length.");
             }
 
-            byte[] byteArray = new byte[readBuffer.Length];
-            _winDevice.TransferFullDuplex(writeBuffer.ToArray(), byteArray);
-            byteArray.CopyTo(readBuffer);
+            if (_isInverted)
+            {
+                byte[] byteArray = new byte[readBuffer.Length];
+                ReverseByte(byteArray);
+                _winDevice.TransferFullDuplex(writeBuffer.ToArray(), byteArray);
+                byteArray.CopyTo(readBuffer);
+                ReverseByte(readBuffer);
```

Comment:
similar comment as with Write, it can be greatly simplified since we always use array anyway